### DISCUSSION TITLE
Do I need to use the ota_data_initial.bin?

### DISCUSSION THREAD

**Discussion by CorrePlayer** [darthcloud/BlueRetro#921] [Q&A]
I'm using a RetroScaler BlueRetro
The bitfunx site doesn't show the ota_initial_bin, do I use it to update or not?

**Comment by darthcloud** [comment]
using OTA you only use the BlueRetro_XXX.bin file.

For USB just follow my instruction as-is (use it)

**Comment by CorrePlayer** [comment]
Yeah, but in what frequency i flash it?
Bootloader is 0x1000, partition table is 0x8000, what should be for ota inicial?
(Mine's USB)

**Reply by darthcloud** [reply]
As in the instructions:
https://github.com/darthcloud/BlueRetro/wiki/Flashing-firmware-Windows-10

0xD000
